packages/main/src/themes/TableRowBase.css

```diff
@@ -20,6 +18,17 @@
 	clip-path: inset(var(--sapContent_FocusWidth)); /* focus outline should not overlap sticky cells */
 }
 
+::slotted(ui5-table-cell:first-of-type),
+::slotted(ui5-table-header-cell:first-of-type),
+#popin-cell {
+	padding-left: var(--_ui5_first_table_cell_horizontal_padding);

```

Comment:
there is no RTL mirroring so padding-inline-start

Reply:
also i guess tag names should not be used because of micro frontend frameworks but attribute selectors,  probably you used it because of the type pseudo class. 

 can we maybe use the [slot=default-1]?